lib/note-list/index.tsx

```diff
@@ -301,23 +386,26 @@ export class NoteList extends Component<Props> {
       onEmptyTrash,
       onPinNote,
       searchQuery,
-      selectedNoteId,
       showTrash,
       tagResultsFound,
     } = this.props;
+    const { selectedIndex: highlightedIndex } = this.state;
 
     const compositeNoteList = createCompositeNoteList(
       notes,
       searchQuery,
       tagResultsFound
     );
 
+    const specialRows = compositeNoteList.length - notes.length;
+
     const renderNoteRow = renderNote(compositeNoteList, {
       searchQuery,
+      highlightedIndex:
+        highlightedIndex !== null ? highlightedIndex + specialRows : null,
```

Comment:
This is okay but it might be clearer to make it a variable (maybe instead of `specialRows`)